docs/docs/cli.md

```diff
@@ -423,6 +423,12 @@ This command locks (without installing) the dependencies specified in `pyproject
 poetry lock
 ```
 
+To avoid updating pinned dependencies, you can use the `--no-update` option.
```

Comment:
I think it would be better to match the pattern used in all the other commands, and add an "Options" section with a line for `--no-update`.